README.md

```diff
@@ -81,6 +81,11 @@ cd docs
 python example_projectq_grover.py
 ```
 
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/QuTech-Delft/quantuminspire.git/master?filepath=%2Fdocs)
+
+Another way to browse and run the available notebooks is by clicking the 'launch binder' button above.
+If you are experiencing delays, please check the 'Binder' system status. The service may be down.
```

Comment:
What, or where, is the binder system status? this one: https://grafana.mybinder.org/d/3SpLQinmk/1-overview?refresh=1m&orgId=1 ?

Reply:
I'll remove this line about system status.